### PR TITLE
select the subreddit from the env file

### PR DESCRIPTION
**Pull request by fabiand93**

### CODE REVIEW

File: reddit/askreddit.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def get_askreddit_threads():
         username=os.getenv("REDDIT_USERNAME"),
         password=os.getenv("REDDIT_PASSWORD"),
     )
-    askreddit = reddit.subreddit("askreddit")
+    askreddit = reddit.subreddit(os.getenv("REDDIT_SUBREDDIT"))
```
Comment (SpaYco) [inline reddit/askreddit.py:24]: let's keep `askreddit` as a default value

File: README.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ These videos on TikTok, YouTube and Instagram get MILLIONS of views across all p
 1. Clone this repository
 2. Rename `.env.template` to `.env` and replace all values with the appropriate fields. To get Reddit keys (**required**), visit [the Reddit Apps page.](https://www.reddit.com/prefs/apps) TL;DR set up an app that is a "script". Copy your keys into the `.env` files.
 3. Run `pip3 install -r requirements.txt`
+4. Run `playwright install`
```
Comment (SpaYco) [inline README.md:36]: _I'm not familiar with Python so ignore this comment if it doesn't make sense_

Do we need to install this in order for the app to work? from looking at their page this is meant for testing, if this is optional then we should say that it is, and also explain what this does in a super short testing (ex: use this for testing)

File: README.md [hunk 3]
```diff
@@ -45,6 +46,6 @@ I have tried to simplify the code so anyone can read it and start contibuting at
 
 - [ ] Allowing users to choose a reddit thread instead of being randomized.
 - [ ] Allowing users to choose a background that is picked instead of the Minecraft one.
-- [ ] Allowing users to choose between any subreddit.
+- [✅] Allowing users to choose between any subreddit.
```
Comment (SpaYco) [inline README.md:49]: out of the scope of this PR, but I think being able to pull from multiple subreddits at once would also be great! WDYT?

**Review by SpaYco — CHANGES_REQUESTED** [review]
hey @fabiand93, great work, I left some comments :)

**Comment by elebumm** [comment]
@fabiand93 Great PR! The last pr before this one did the same functionality, so sadly I have to close this one :( I think this is a great feature though.

**Pull request closed without merging**